lib/gtype.mli

```diff
@@ -53,6 +53,9 @@ val of_protocol : Syntax.global_protocol -> t
 val global_t_of_module : Syntax.scr_module -> global_t
 (** Turn scribble module (from the parser) into a global type *)
 
+val global_t_of_ast : Syntax.scr_module -> global_t
```

Comment:
^ considering that share the same signature

Reply:
I can't remember the timeline exactly, but initially I did not apply normalisation and I did not realise that I had to deal with recursion as nested protocols for the code generation. It may be that I forgot to merge them/remove one of them after implementing the second function